sdk/c/src/ydc-c.c

```diff
@@ -0,0 +1,106 @@
+/**************************************************************
+ * Copyright (c) 2018-2020,Hangzhou Rokid Tech. Co., Ltd.
+ * All rights reserved.
+ *
+ * FileName: ydc-c.c
+ * Description: Dynamic library interface
+ *
+ * Date  :	2019.06.18
+ * Author:  zijiao.wang@rokid.com
+ * Modification: Init version
+ *
+ **************************************************************/
+
+#include <stdlib.h>
+#include <stdint.h>
+#include <unistd.h>
+#include <uv.h>
+#include <pthread.h>
+#include <cjson/cJSON.h>
```

Comment:
At YodaOS 8.0, we use https://github.com/nlohmann/json as our default library to replace CJSON, it's better to replace here, too.

Reply:
Just resolved this, https://github.com/nlohmann/json is for C++, and we use CJSON for pure C :)